test/fixtures/programmatic/invocation/cjsasync.cjs

```diff
@@ -0,0 +1,14 @@
+'use strict';
```

Comment:
I'd use `-` separator. theoretically `async.cjs` feels as best name, but I understand that it won't work for testing when there's `async.js`, so maybe `async-cjs.cjs` ?